#
# Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
#
# This patch fixes also:
# 16222870 - xkbcomp warns about <RALT> having 2 symbols
#
--- a/symbols/pc	2012-02-06 14:36:51.420565110 +0100
+++ b/symbols/pc	2012-02-10 12:57:06.622918884 +0100
@@ -44,11 +44,14 @@
     modifier_map Mod5   { <LVL3>, <MDSW> };
 
     key <ALT>  {	[ NoSymbol, Alt_L	]	};
-    include "altwin(meta_alt)"
+    key <RALT> {
+    type[Group1]="ONE_LEVEL",
+    symbols[Group1] = [ ISO_Level3_Shift ]
+    };
+    modifier_map Mod5   { ISO_Level3_Shift  };
+    key <LALT> {        [ Alt_L,  Meta_L  ] };
+    modifier_map Mod1   { Alt_L };
 
-    key <META> {	[ NoSymbol, Meta_L	]	};
-    modifier_map Mod1   { <META> };
-
     key <SUPR> {	[ NoSymbol, Super_L	]	};
     modifier_map Mod4   { <SUPR> };
 
@@ -56,10 +59,11 @@
     modifier_map Mod4   { <HYPR> };
     // End of modifier mappings.
 
-    key <OUTP> { [ XF86Display ] };
-    key <KITG> { [ XF86KbdLightOnOff ] };
-    key <KIDN> { [ XF86KbdBrightnessDown ] };
-    key <KIUP> { [ XF86KbdBrightnessUp ] };
+// The following create some duplicate keycodes in Solaris
+//    key <OUTP> { [ XF86Display ] };
+//    key <KITG> { [ XF86KbdLightOnOff ] };
+//    key <KIDN> { [ XF86KbdBrightnessDown ] };
+//    key <KIUP> { [ XF86KbdBrightnessUp ] };
 };
 
 hidden partial alphanumeric_keys
